### ISSUE TITLE
Develop Record 01

### ISSUE THREAD

**Issue by nanguoyu** [nanguoyu/SemiFlow#16]
Hi, I am Dong. There is a problem about validation.

In the [mnist_mlp example](https://github.com/nanguoyu/SemiFlow/blob/master/example/mnsit_mlp.py), the result shows that the loss curve of validation is lower than training curve. Even at the first epoch, this model got better result in validation data. I guess that there are two reasons,

- This model is overfitting
- After updating parameters at first time, the model can reduce the loss value to a low number. Then the validation is performed and gets low value
